go.mod

```diff
@@ -26,6 +26,7 @@ require (
 	github.com/multiformats/go-multiaddr v0.8.0
 	github.com/multiformats/go-multiaddr-dns v0.3.1
 	github.com/olekukonko/tablewriter v0.0.5
+	github.com/onsi/gomega v1.26.0
```

Comment:
This is a new dependency, which I would usually try to avoid in a PR like this.  But, gomega provides some very nice asynchronous polling primitives that reduce the code complexity of dealing with external processes.  In particular waiting for the process to become ready by watching the log output can be accomplished safely and without long sleeps in a single line (I'll note this below).

If we're opposed to bringing in this dependency, it can be written out, but will require more code.